elements/howto-tabs/howto-tabs.js

```diff
@@ -27,13 +27,26 @@
   };
 
   // To avoid invoking the parser with `.innerHTML` for every new instance, a
-  // template for the contents of the ShadowDOM is  is shared by all
+  // template for the contents of the ShadowDOM is shared by all
   // `<howto-tabs>` instances.
-  const shadowDOMTemplate = document.createElement('template');
-  shadowDOMTemplate.innerHTML = `
-    <slot name="tab"></slot>
-    <slot name="panel"></slot>
-  `;
+  const template = Object.assign(
```

Comment:
I feel like this is actually more code than the way I was doing it :D what's the upside of using object.assign?